Exercism/Gigasecond/Gigasecond.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Exercism.Gigasecond
+{
+    public static class Gigasecond
+    {
+        public static DateTime addGigasecond(DateTime birthDate) => birthDate.AddSeconds(1000000000);
```

Comment:
Also `birthDate` isn't a great variable name since it has nothing to do with birth dates